extensions/yaml_support/_parser/base_parser.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABCMeta, abstractmethod
```

Comment:
Are there some unit/integration tests we could use for this new feature?

Reply:
Hello, Jimmy Thank you for the review,, I am searching what are some proper ways to test a parser and I would like to have some guidance on that.

Reply:
As we have discussed earlier, you can use this as an example for testing: https://github.com/facebookresearch/labgraph/blob/main/labgraph/zmq_node/tests/test_zmq_node.py

Reply:
Please also add in Facebook license header and python header in all the relevant files:
#!/usr/bin/env python3
# Copyright 2004-present Facebook. All Rights Reserved.

Reply:
Done